Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG GOLANG_VERSION=1.22.8
 ARG CUDA_VERSION_11=11.3.1
 ARG CUDA_VERSION_12=12.4.0
-ARG ROCM_VERSION=6.1.2
+ARG ROCM_VERSION=6.2.4
```

Comment:
```suggestion
ARG ROCM_VERSION=6.3.2
```

Reply:
Uhm.. Maybe that needs to be reported here. https://github.com/ROCm/ROCm-docker/blob/master/dev/Dockerfile-centos-7

Since they didn't publish new images for multiple months!

Reply:
Nice to see to move to v6.3.2 indeed by using almalinux.